src/App.tsx

```diff
@@ -15,71 +16,92 @@ export const goods = [
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  handleClearSelectedGood = () => {
+    this.setState({ selectedGood: '' });
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  render() {
+    const { selectedGood } = this.state;
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+    return (
+      <main className="section container">
+        {!selectedGood
+          ? (
+            <h1 className="title">No goods selected</h1>
+          ) : (
+            <h1 className="title is-flex is-align-items-center">
+              {`${selectedGood} is selected`}
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={this.handleClearSelectedGood}
+                aria-label="ClearButton"
+              />
+            </h1>
+          )}
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+        <table className="table">
+          <tbody>
+            {goods.map((good) => {
+              const isCheckedGood = selectedGood === good;
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+              return (
+                <tr
+                  data-cy="Good"
+                  className={classNames({
+                    'has-background-success-light': isCheckedGood,
+                  })}
+                >
+                  {isCheckedGood
+                    ? (
+                      <td>
+                        <button
+                          data-cy="RemoveButton"
+                          type="button"
+                          className="button is-info"
+                          onClick={this.handleClearSelectedGood}
+                        >
+                          -
+                        </button>
+                      </td>
+                    ) : (
+                      <td>
+                        <button
+                          data-cy="AddButton"
+                          type="button"
+                          className="button"
+                          onClick={() => {
+                            this.handleSelectedGood(good);
+                          }}
+                        >
+                          +
+                        </button>
+                      </td>
+                    )}
```

Comment:
it could be simplified
```suggestion
                  <td>
                    <button
                      data-cy={isCheckedGood ? 'RemoveButton' : 'AddButton'}
                      type="button"
                      className={isCheckedGood ? 'button is-info' : 'button'}
                      onClick={() => {
                        this.handleSelectedGood(good);
                      }}
                    >
                      {isCheckedGood ? '-' : '+'}
                    </button>
                  </td>
```